docs/tutorials/infrastructure/kne/srl-with-kne/srl-with-oc-services.md

```diff
@@ -0,0 +1,255 @@
+
+[oc-doc]: https://documentation.nokia.com/srlinux/22-6/SR_Linux_Book_Files/SysMgmt_Guide/data-models.html#openconfig-ov
+
+By default, Nokia SR Linux uses native [YANG models](../../../../yang/yang.md). Openconfig YANG models are already enabled in the configuration file used in this lab.
+
+For completeness, the below section shows how to enable Openconfig via different management interfaces.
+
+=== "CLI"
+    ```
+    --{ running }--[  ]--
+    A:srl# enter candidate
+
+    --{ candidate shared default }--[  ]--
+    A:srl# system management openconfig admin-state enable
+    
+    --{ * candidate shared default }--[  ]--
+    A:srl# commit stay 
+    All changes have been committed. Starting new transaction.
+    ```
+=== "Config file"
+    ```json
+    "srl_nokia-system:system": {
+      "management": {
+        "srl_nokia-openconfig:openconfig": {
+          "admin-state": "enable"
+        }
+      },
+    // other system containers
+    }
+    ```
+
+### gNMI
+
+<small>[:octicons-book-16: gNMI docs][gnmi-doc]</small>
+
+[gnmi-doc]: https://documentation.nokia.com/srlinux/22-6/SR_Linux_Book_Files/SysMgmt_Guide/gnmi-interface.html
+
+gNMI service is enabled over port `57400` in the configuration files used with this lab.
+
+<!-- For completeness, the below section shows how to enable gNMI via different management interfaces.
+
+=== "CLI"
+    ```
+    --{ running }--[  ]--
+    A:srl# enter candidate
+
+    --{ candidate shared default }--[  ]--
+    A:srl# system management openconfig admin-state enable
+    
+    --{ * candidate shared default }--[  ]--
+    A:srl# commit stay 
+    All changes have been committed. Starting new transaction.
+    ```
+=== "Config file"
+    ```json
+    "srl_nokia-system:system": {
+      "management": {
+        "srl_nokia-openconfig:openconfig": {
+          "admin-state": "enable"
+        }
+      },
+    // other system containers
+    }
+    ``` -->
+
+??? "Example"
+    gNMI service can be tested using [gnmic](https://gnmic.kmrd.dev) cli client.
+
+    ```bash
+    ❯ gnmic -a 172.18.0.50:57400 -u admin -p admin --skip-verify capabilities
+    gNMI version: 0.7.0
+    supported models:
+      - urn:srl_nokia/aaa:srl_nokia-aaa, Nokia, 2022-06-30
+      - urn:srl_nokia/aaa-password:srl_nokia-aaa-password, Nokia, 2022-06-30
+      - urn:srl_nokia/aaa-types:srl_nokia-aaa-types, Nokia, 2021-11-30
+      - urn:srl_nokia/acl:srl_nokia-acl, Nokia, 2022-06-30
+    -- snip --
+    ```
+
+### gNOI
+
+<small>[:octicons-book-16: gNOI docs][gnmi-doc]</small>
+
+[gnoi-doc]: https://documentation.nokia.com/srlinux/22-6/SR_Linux_Book_Files/SysMgmt_Guide/gnoi_interface.html
+
+gNOI service is enabled automatically once gNMI service is operational. gNMI and gNOI share the same port `57400`.
+
+??? "Example"
+    gNOI service can be tested using [gnoic](https://gnoic.kmrd.dev) cli client.
+
+    ```bash
+    ❯ gnoic -a 172.18.0.50:57400 --skip-verify -u admin -p admin file stat --path /etc/os-release
```

Comment:
likewise 9337

and I got another error:

```
alexmasi@alexmasi:~$ gnoic -a 192.168.11.50:9337 --skip-verify -u admin -p admin file stat --path /etc/os-release
WARN[0000] "192.168.11.50:9337" could not lookup hostname: lookup 50.11.168.192.in-addr.arpa. on 127.0.0.1:53: no such host 
ERRO[0010] "192.168.11.50:9337" File Stat failed: context deadline exceeded 
+-------------+------+--------------+------+-------+------+
| Target Name | Path | LastModified | Perm | Umask | Size |
+-------------+------+--------------+------+-------+------+
+-------------+------+--------------+------+-------+------+
Error: there was 1 error(s)
```

Reply:
like with gnmi, I don't have an issue with gnoi over 9337 either...

PS. the warn message is expected (this will be fixed in gnoic, but not related to this PR)